en/stacks/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: What is Ubuntu Stacks?
+---
+
+# What is Ubuntu Core Stacks?
+
+Ubuntu Core Stacks is a set of snap packages that offer system-level functionality that
+enables important core system-level features not offered by the base Ubuntu Core
+system. With little to no additional effort, these snaps can be dropped in place
+on one or more new or existing systems bringing significant new value to these systems.
+These snaps have been well tested in various commercial offerings and are
+actively maintained by an internal Canonical team.
+
+Please note that some additional integration work to these snaps are required if
+new application of these snaps diverges significantly from existing applications.
```

Comment:
It is overloaded indeed, however that's ok in this context. For example, if someone reads it as an app like on a phone vs applying a snap to something, it still gives the same context. What I meant here though is if someone wants to use NetworkManager for example but they want it to manage ethernet connections, they'd need to do the work. It's not a great example since we'll be doing it, but it makes my point.